sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -1,28 +1,65 @@
 package uk.ac.wellcome.models.work.internal
 
+import uk.ac.wellcome.models.work.text.TextNormalisation._
+
 sealed trait AbstractRootConcept {
   val label: String
 }
 
 sealed trait AbstractConcept extends AbstractRootConcept
 
 case class Concept(label: String) extends AbstractConcept
+object Concept {
```

Comment:
minor: elsewhere we've used `case object` to highlight that these are companion objects, although I confess I'm not entirely sure what the distinction means!

Reply:
I don't think case is useful for object. The only effect it has is addin methods like copy, equals, hash and so on. It makes little sense for objects and I generally prefer to leave it out